### ISSUE TITLE
Configure page for 500 errors

### ISSUE THREAD

**Issue by jthingelstad** [WikiApiary/WikiApiary#147]
Hopefully not needed often, but if bad things happen there should be an error page instead of a null response with a blank white page.

**Comment by kghbln** [comment]
Yeah, do not forget to include a note that uptime is monitored and that no specific steps need to be taken by the visitors. Just sit tight. :)
